avaframe/ana5Utils/distanceTimeAnalysis.py

```diff
@@ -0,0 +1,653 @@
+""" functions to convert to a different coordinate system and
+    produce a range-time diagram from simulation results
+    options: 1) range-time diagram from radar's field of view capped to this
+             2) thalweg-time diagram from beta point of view along avalanche path
+"""
+
+
+import numpy as np
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+import logging
+import pathlib
+
+# Local imports
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+import avaframe.in2Trans.ascUtils as IOf
+import avaframe.out3Plot.outTopo as oP
+import avaframe.in3Utils.geoTrans as gT
+import avaframe.out3Plot.outDistanceTimeAnalysis as dtAnaPlots
+import avaframe.ana3AIMEC.aimecTools as aT
+import avaframe.out3Plot.plotUtils as pU
+import avaframe.com1DFA.com1DFA as com1DFA
+import avaframe.in3Utils.fileHandlerUtils as fU
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def getRadarLocation(cfg):
+    """ fetch radar location and direction of of field of view coordinates
+
+        Parameters
+        ------------
+        cfg: configparser object
+            configuration settings
+
+        Returns
+        --------
+        radarFov: numpy array
+            array with x and y coordinates of radar location and point in direction of field of view
+
+    """
+
+    # fetch coordinate info
+    locationInfo = cfg['GENERAL']['radarLocation'].split('|')
+
+    # set radar location and point in direction of field of view as list
+    radarFov = np.asarray([[float(locationInfo[0]), float(locationInfo[1])], [float(locationInfo[2]),
+        float(locationInfo[3])]])
```

Comment:
Please catch and add an log.error on non-existent location info in case of TTdiagram=false

Reply:
added an error if len(locationInfo) != 4